docs/contributing.rst

```diff
@@ -1,2 +1,37 @@
 Contributing
 ============
+
```

Comment:
It would be great to broader options for engagement---not solely focused on software **development.** Firstly, just create an issue if you notice something that's not expected (or email for support).  People might be interested in creating an app rather than contributing HS core.